### PR TITLE
[MIRROR] Sec Resprite Fixes [MDB IGNORE]

### PR DESCRIPTION
**Pull request by SkyratBot**
Original PR: https://github.com/tgstation/tgstation/pull/70030
--------------------

## About The Pull Request

Initially meant to be a quick fix for the black splotches on the sec resprite. Upon further inspection found multiple consistency issues, missing inhand resprites, awkward designs, and poor quality assurance. This PR fixes those issue.
dreamseeker_8Bq1ehS6CJ
Sec helmets have been slightly reduced in size.
Normal sec helmet chin strap has returned.
Alt sec helmet will keep the no-chinstrap look.
Sec wintercoat hood now looks like a hood for a wintercoat.
New sec armor has been slightly reduced in size.
Black splotches on sec pants have been removed.
All sec personnel shoulder patches are now on both arms.
Sec uniform belt buckle is now black for a better distinction between sec and warden.
Fixes secskirt skirt size.
Fixes gold trim on hos uniforms.
Adds sec helmet inhands to match the new sprites.
Adds sec wintercoat inhands to match the new sprites.
Adds sec helmet obj icons to match the new sprites.
Fixes squished secskirt obj icon.
Things i probably forgot.

## Why It's Good For The Game

Department uniforms, especially sec, should be clear and concise. Red and black chaps are overcomplicated and unnecessary. Consistency is all things good and pure.

## Changelog

:cl:
imageadd: fixes previous sec resprite issues, replaces sec helmet and sec wintercoat inhands, replaces sec helmet obj icon
/:cl:

### CODE REVIEW

**Comment by Ebin-Halcyon** [comment]
😔

**Comment by ghost** [comment]
Another one :(

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -107,8 +107,13 @@
 
 /obj/structure/closet/secure_closet/security/PopulateContents()
 	..()
+<<<<<<< HEAD
 	new /obj/item/clothing/suit/armor/vest/alt/sec(src) //SKYRAT EDIT CHANGE - Original: /obj/item/clothing/suit/armor/vest(src) (UPDATE THIS WHENEVER TG FINALLY DOES)
 	new /obj/item/clothing/head/security_cap(src) //SKYRAT EDIT ADDITION
+=======
+	new /obj/item/clothing/suit/armor/vest(src)
+	new /obj/item/clothing/suit/armor/vest/alt/sec(src)
+>>>>>>> d1582bf1b41 (Sec Resprite Fixes (#70030))
```
Comment (ghost) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]:
```suggestion
//	new /obj/item/clothing/suit/armor/vest(src) //SKYRAT EDIT REMOVAL
	new /obj/item/clothing/suit/armor/vest/alt/sec(src)
	new /obj/item/clothing/head/security_cap(src) //SKYRAT EDIT ADDITION
```
Don't need TG's plain vest, our security have special ones

**Review by ghost — COMMENTED** [review]
Digi icons will need updating but it's lower priority. After fixing conflict, LGTM

**Pull request merged**